frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/helpers.ts

```diff
@@ -1,10 +1,16 @@
 import {
   IAppStoreApp,
+  ISoftwarePackage,
   ISoftwareTitleDetails,
   isSoftwarePackage,
 } from "interfaces/software";
 import { DEFAULT_EMPTY_CELL_VALUE } from "utilities/constants";
 
+const mergePackageStatuses = (packageStatuses: ISoftwarePackage["status"]) => ({
+  installed: packageStatuses.installed,
+  pending: packageStatuses.pending_install + packageStatuses.pending_uninstall,
```

Comment:
How is this used? Is there someplace where we are checking for the potential string `"pending_installpending_uninstall"`? Are we counting on the backend to ensure that never happens?

Reply:
If I'm understanding your question, those fields actually contain _counts_ of hosts in their respective statuses, not strings representing those statuses, so this gives us the total number of hosts pending install or pending uninstall.